ops/scripts/geth-version-checker.sh

```diff
@@ -7,7 +7,7 @@ MONOREPO_DIR=$(cd "$SCRIPTS_DIR/../../" && pwd)
 GETH_VERSION="v$(geth version | grep '^Version:' | awk '{print $2}')"
 
 # Read the version from the versions file
-EXPECTED_GETH_VERSION=$(jq -r .geth < $MONOREPO_DIR/versions.json)
+EXPECTED_GETH_VERSION=$(jq -r .geth < "$MONOREPO_DIR"/versions.json)
 
```

Comment:
I think its weird to have `"$MONOREPO_DIR"/versions.json` over `"$MONOREPO_DIR/versions.json"`, maybe this is a personal style nit but it just reads more obvious to me

Reply:
Yea I think your version is a little cleaner too. It seems we're currently inconsistent about this in the rest of the codebase though, so not sure if different devs have different style preferences